app/command_utils.cc.template

```diff
@@ -83,7 +83,7 @@ std::string GetCommandName(int command_id) {
     // 2. The & is in brackets, and defines a shortcut key not in the string, in
     //    which case we remove the whole match. For example "Install (&K)PWA".
     auto remove = base::StartsWith(match, "(") ? match : "&";
```

Comment:
Q. Can we just use match without checking `base::StartsWith()`?

Reply:
Nah, because in the case of `&JavaScript` we just want to remove the `&`, but in the case of `(&J)Do the thing` we want to remove `(&J)`.